RELNOTES.md

```diff
@@ -2,6 +2,55 @@
 
 © Copyright EnterpriseDB UK Limited 2015-2023 - All rights reserved.
 
+## v23.24 (unreleased)
+
+### Notable changes
+
+- TPA-499 Change default ansible version to community ansible
+
+  `tpaexec setup` now defaults to using community ansible rather than
+  2ndQuadrant ansible. The option `--use-2q-ansible` can be used to
+  force the use of 2ndQuadrant ansible, which is now deprecated and will
+  be removed in a future release.
```

Comment:
We should link to the detailed section in docs that explains the impact from this change